website/docs/docs/dbt-versions/release-notes/11-Feb-2023/feb-ide-updates.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "IDE updates and fixes"
+id: "feb-ide-updates"
+description: "Feb 2023: We've enhanced the IDE by adding custom node colors in the DAG, ref autocomplete, double-click files to rename them, add link to repo from the branch name, enabled syntax highlighting for jinja, improve file tree render time, and more."
+sidebar_label: "Update and fixes: IDE"
+tags: [Mar-1-2023, IDE]
+---
+
+To continue improving our [Cloud IDE](https://docs.getdbt.com/docs/get-started/develop-in-the-cloud) experience, the dbt Labs team worked on fixing bugs, increasing reliability, and adding new features ✨.
+
+Learn more about the [February changes](https://getdbt.slack.com/archives/C03SAHKKG2Z/p1677605383451109). 
+
+## New features 
+
+- Support for custom node colors in the IDE DAG visualization
+- Autosave prototype is now available under feature flag. [Contact](mailto:cloud-ide-feedback@dbtlabs.com) the dbt Labs IDE team to try this out
+- Ref autocomplete includes models from seeds and snapshots
+- Prevent menus from getting cropped (git controls dropdown, file tree dropdown, build button, editor tab options)
+- Additional option to access the file menu by right-clicking on the files and folders in the file tree
+- Rename files by double-clicking on files in the file tree and the editor tabs
+- Right-clicking on file tabs has new options and will now open at your cursor instead of in the middle of the tab
+- The git branch name above **Version Control** links to the repo for specific git providers
```

Comment:
I would add a short note in parenthesis that this is technically only available for MT github and gitlab

Reply:
gotcha thanks! to clarify, you mean US MT? or all MT instances (emea, us, apac)?

Reply:
stefan confirmed its all mt

Reply:
all of MT yes!